packages/server/graphql/mutations/signUpWithPassword.ts

```diff
@@ -74,7 +74,7 @@ const signUpWithPassword = {
       if (!nickname || !domain) {
         return {error: {message: 'Invalid email'}}
       }
-      const verificationRequired = await isEmailVerificationRequired(domain)
+      const verificationRequired = await isEmailVerificationRequired(email, dataLoader)
```

Comment:
we kinda did domain locking in the past. if an org chose to lock their domain, you couldn't create a parabol account unless you verified your email first. since this PR does that & so much more, i changed the logic to share the same DB table.